CHANGELOG

```diff
@@ -1,3 +1,67 @@
+==== 2.2 beta 2018-09-07 ====
```

Comment:
Nit: this isn't really a changelog -- it's a release announcement (which it's great to have a draft of!) but the goal of the changelog is to be a succinct list of what changed. 

Compare to:
```
* Changed: Switch to Qt 5 (better HiDPI support).
* Removed: Manual display scaling option in the preferences.
* Added: Mode switch for mixing effects (Dry/Wet and Dry+Wet) [see manual](http://link-to-manual/).
* Added: LV2 Effect Plugin support on GNU/Linux and macOS.
* Added: Option to adjust opacity of waveform beatgrid lines.
* Added: Option to adjust position of the waveform play position indicator.
```
It's much quicker to read a bulleted list of changes than long form text descriptions.

https://keepachangelog.com/en/1.0.0/

It'd be nice if this file were markdown so we could add formatting and links to the manual.

Reply:
Good point. I changed it to a bulleted list.